### ISSUE TITLE
Appliance type 'Dryer' not currently supported

### ISSUE THREAD

**Issue by smhex** [thoukydides/homebridge-homeconnect#107]
#### Description of Issue
When updating my instance to version 0.26.0 of the plugin I got the following error message:
```
[9/23/2022, 6:17:34 PM] [HomeConnect] Appliance type 'Dryer' not currently supported
```

It is not longer available in HomeKit. It was working fine with plugin version 0.25.0.

#### Plugin Version
0.26.0

#### Home Connect Appliance(s)
Bosch dryer

#### Diagnostic Checks
Reverting back to the previous version (0.25.0) solved the problem.

#### Log File

```
[9/23/2022, 6:17:34 PM] [HomeConnect] Appliance type 'Dryer' not currently supported
[9/23/2022, 6:17:34 PM] [HomeConnect] [Waschmaschine] Bosch Washer (E-Nr: WAV28G43/04)
[9/23/2022, 6:17:34 PM] [HomeConnect] [Geschirrspüler] Bosch Dishwasher (E-Nr: SMV8YCX01E/17)
[9/23/2022, 6:17:34 PM] [HomeConnect] Removing accessory 'Trockner'
```

I guess that it is somehow related to the latest changes in lib/appliance_cleaning.js.

Thanks for looking into it!
smhex

**Comment by thoukydides** [comment]
Oops. Sorry about that.
Should be fixed in v0.26.1.

**Comment by smhex** [comment]
Thanks, works again as expected!
